src/select-state.js

```diff
@@ -0,0 +1,69 @@
+import { createSelector } from 'reselect';
```

Comment:
> Would we want to place these input selectors contextually within their respective selector files as opposed to having them all within a single file?

Yes, we probably should!  I put all the selectors that are nothing more than pulling a single prop out of state in a single file for convenience while developing, but we probably should disperse all these "selectors" into their respective modules.

> Another assuming I had in reviewing was that these changes will supersede the existing selector approach of having a global selector object, correct?

Yep!  The global selector object is still there since the conversion isn't 100% complete yet, but my goal here is to get rid of the "selector object" entirely and replace it with selector functions -- since a selector is, after all, by definition a function and (to me) it's weird and confusing to have "selectors" which are just sitting in a global object and looking for all the world like a second state object.